ui/src/components/layouts/HalfMapLayout/NewMap.tsx

```
@@ -0,0 +1,416 @@
+              //   },
+              // }}
+            >
+              <Tooltip
+                className="custom-tooltip"
+                direction="top"
+                offset={[0, 0]}
+                opacity={1}
+                permanent={false}
+              >
+                <div
+                  dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)